src/mentors.json

```diff
@@ -1595,11 +1595,11 @@
-      "fullBio": "Having worked in the media industry for three decades, Serene Goh is currently the founder of O\u2019Chim, a bakeshop and solutions consultancy. Launched in 2018, it aims to help clients shape their stories, and establish their presence for maximum good. She believes that human-centered narratives define civilisations, and is driven to innovate content that tells a story. Her career in media is marked by strategies that combine pragmatism with business development to shape stories across formats and platforms. Specialising in reaching millennial and young audiences, Serene led The Straits Times young reader department for 12 years. She drove national events, engagement, literacy campaigns, including the RHB-Straits Times National Spelling Championship for primary schools, the National News Media Competition for secondary schools, the National Current Affairs Quiz for junior colleges, and the MCCY-Straits Times Idea Jam, a national design thinking campaign, for tertiary students. In her care, the department notched at least 14 global awards recognising its excellence. Additionally, she was appointed to head Content Lab in 2016, Singapore Press Holdings\u2019 content marketing unit at its Integrated Marketing Division, unifying its print and digital paid-content teams. Through coaching and people development, her team earned three international awards for achievement in strategic marketing over two years.",
+      "fullBio": "Having worked in the media industry for three decades, Serene Goh is currently the founding Director of Brango Pte. Ltd., a solutions consultancy. She also founded O'Chim, a bakeshop. Launched in 2018, it aims to help clients shape their stories, and establish their presence for maximum good. Serene believes that human-centered narratives define civilisations, and is driven to innovate content that tells a story. Her career in media is marked by strategies that combine pragmatism with business development to shape stories across formats and platforms. Specialising in reaching millennial and young audiences, Serene led The Straits Times young reader department for 12 years. She drove national events, engagement, literacy campaigns, including the RHB-Straits Times National Spelling Championship for primary schools, the National News Media Competition for secondary schools, the National Current Affairs Quiz for junior colleges, and the MCCY-Straits Times Idea Jam, a national design thinking campaign, for tertiary students. In Serene's care, the department notched at least 14 global awards recognising its excellence. Additionally, she was appointed to head Content Lab in 2016, Singapore Press Holdings' content marketing unit at its Integrated Marketing Division, unifying its print and digital paid-content teams. Through coaching and people development, Serene's team earned three international awards for achievement in strategic marketing over two years.",
       "imageUrl": "/profile_images/SereneGoh.jpg"
```

Comment:
Serene's profile picture seems to be removed. The new profile picture shld be uploaded instead, see the spreadsheet.

Reply:
Is it there.

Reply:
Oops lol I mistook the decrease in size to be a delete.